.github/workflows/std-release.yaml

```diff
@@ -9,7 +9,8 @@ jobs:
     if: startsWith(github.event.release.name, '@cardano-sdk/cardano-services@')
     uses: ./.github/workflows/std.yml
     with:
-      deploy-staging-preprod: true
+      # Only deploy staging-* on demand, otherwise they don’t differ from dev-*:
```

Comment:
Does this need to be explicit? Should we not just remove it from the list here? I don't feel strongly either way, but it seems a bit inconsistent since `dev-preview` is not here

Reply:
This is because `dev-preview` [is not in `workflow_call: inputs:`](https://github.com/input-output-hk/cardano-js-sdk/blob/e177883e406baafbac3aeb8802e8f8c2aa022e26/.github/workflows/std.yml#L26-L36).

Maybe for consistency, we should add it but set `required: false` for both `deploy-dev-preview` and `deploy-staging-preprod`, and then don't list them explicitly here?

Maybe set `required: false` for all, which would make `false` their default :)

Reply:
Or remove `deploy-staging-preprod` from the `workflow_call: inputs:`? 🤔 All of these are equivalent functionality-wise, but which is the most readable?

Reply:
My first though is the correct semantics seems to be:

> remove deploy-staging-preprod from the workflow_call: inputs:

... as this says the workflow should only be invoked with a dispatch, right? However, if that's the case, why doesn't `dev-preview` need to be listed in `workflow_call: inputs`?

Reply:
Because `dev-preview` is deployed on each merge to `master` (or with a manual `workflow_dispatch:`), so it's most probably already been deployed once a release happens.

But I can add it to release's `workflow_call:` for consistency, it won't hurt

Reply:
I’ll look at it again in more depth. I’m more concerned about semantics here, so just do the minimum to make it work. If that’s what you already have, please ignore my comment 😬

Reply:
Okay, so I think the minimal change regarding the diff is an explicit `s/true/false/`, which we have now.

I have implemented your comment suggestion below.

I'll be happy to implement the future results of your thinking on this :)